### DISCUSSION TITLE
Discussion Topics for Monday July 12 Engineer Sync

### DISCUSSION THREAD

**Discussion by jimduff-usds** [CDCgov/prime-reportstream#1560] [Q&A]
@ronaldheft-usds
@RickHawesUSDS
@mry-usds
@mauricereeves
@Mr0grog
@joemeree
@carlosfelix2 
@pablocaviglia
@tlosbo
@cbroome 
@td-usds 
@jorg3lopez 
@brick-green-agile6 
@ahay-agile6 

If you have topic you'd like to discuss at the Engineering Sync, add it as a comment here

**Comment by tlosbo** [comment]
History API proposal.

**Comment by MauriceReeves-usds** [comment]
Decoupling the incoming message mechanism from the rest of the app, and using a storage queue to save the incoming work, and then reading from the queue instead.
